charts/postgres-operator/Chart.yaml

```diff
@@ -1,13 +1,18 @@
 apiVersion: v1
 name: postgres-operator
-version: 0.1.0
-appVersion: 1.1.0
+version: 1.2.0
+appVersion: 1.2.0
 home: https://github.com/zalando/postgres-operator
 description: Postgres operator creates and manages PostgreSQL clusters running in Kubernetes
 keywords:
 - postgres
 - operator
+- cloud-native
+- patroni
+- spilo
 maintainers:
+- name: Zalando
+  email: opensource@zalando.de
```

Comment:
I believe maintainers in this case are we aka  "Team ACID @ Zalando <team-acid@zalando.de>"

Reply:
should our team mail really be that visible? I think, it's fine this way.